homeassistant/components/awair/manifest.json

```diff
@@ -6,5 +6,12 @@
   "codeowners": ["@ahayworth", "@danielsjf"],
   "config_flow": true,
   "iot_class": "cloud_polling",
```

Comment:
I'm looking for guidance on what value to set this as the integration will support both cloud polling and local polling after this change.  Ideally on the integrations page it will show the cloud badge when configured to use the cloud api and not show that badge when using the local API. As far as I can tell that badge is driven by the value set in the manifest.

Reply:
local